CMakeLists.txt

```diff
@@ -9,6 +9,11 @@ project(freeopcua)
 
 option(BUILD_CLIENT "Build Client" ON)
 option(BUILD_SERVER "Build Server" ON)
+
+option(SSL_SUPPORT_MBEDTLS "Support rsa-oaep password encryption using mbedtls library " OFF)
```

Comment:
mbedtls seems to be in ubuntu/debian so I propose to thave this option ON as default

Reply:
Well Ubuntu users will be happy, I'm on Debian which lacks mbedtls, so cmake cannot find correct paths. We can try cmake to find mbedtls and use hard-coded paths, if it fails. However, pathse are not really hard-coded, they can be set during cmake invocation to whatever you want, I'm using them this way
```
cmake -DSSL_SUPPORT_MBEDTLS=ON -DSSL_SUPPORT_MBEDTLS_INCLUDE_DIR="$$PWD/../ssl/mbedtls/include" -DSSL_SUPPORT_MBEDTLS_LIB_DIR="$$QF_PROJECT_TOP_BUILDDIR/lib/" $$CMAKE_CC_DEF $$CMAKE_CXX_DEF $$PWD/freeopcua
```

Reply:
mbedtls also on Fedora, just saying.

Reply:
I've found also the Jessie backport, so package is here. Anyway, I have not found way how to test it's existence or paths using cmake or pkg-config.
```
pkg-config --list-all | grep mbedtls 
```
returned nothing on Debian 8

 Fortunatelly on Debian, 8 the library is installed to the standard LIB, INCLUDE locations, so freeopcau can be compiled with mbedtls support without extra paths specified.